examples/android/helloworld/app/.gitignore

```diff
@@ -1 +1,2 @@
 /build
+.cxx/
```

Comment:
Why adding this? Anything producing this directory?

Reply:
Yes, this is cmake output directory. See screenshot below:
![image](https://user-images.githubusercontent.com/149684/92648963-fbe02680-f2f2-11ea-9d01-731c3434af97.png)